packages/tailwindcss/src/utilities.ts

```diff
@@ -2785,6 +2792,699 @@ export function createUtilities(theme: Theme) {
     position: (value) => [gradientStopProperties(), decl('--tw-gradient-to-position', value)],
   })
 
+  if (enableMaskUtilities) {
+    /**
+     * @css `mask-image`
+     */
+
+    staticUtility('mask-none', [['mask-image', 'none']])
```

Comment:
We don't have a test for this utility

Reply:
Oh wait, we do! It is just because we have another `mask-none` defined on line 2920.